packages/stylelint-plugin/src/index.ts

```diff
@@ -21,6 +21,10 @@ const kaizenStylelintRules = {
     true,
     { severity: "warning", disableFixing: false },
   ],
+  "kaizen/use-deprecated-component-library-helpers-scss-imports": [
+    true,
+    { severity: "error", disableFixing: false },
```

Comment:
The severity is "error" because this PR also introduces a breaking change where the old imports dissapear (if the consumer doens't run `--fix` then their codebase won't even compile)